wiki/TypeScript-Migration.md

```diff
@@ -0,0 +1,47 @@
+# TypeScript migration
+
+The following steps are recommended for migrating components to TypeScript.
+
+1. [Pick a component]((https://hashicorp.atlassian.net/browse/HDS-2392)), assign yourself to the task in Jira and add it to the current sprint
+
+2. Rename the backing class from `.js` to `.ts`; commit after this step to make it easier to review the changes
+
+3. Opening the files (`.js` and `.hbs`) in VS Code should show you the linting errors (if you have the [Glint extension](https://marketplace.visualstudio.com/items?itemName=typed-ember.glint-vscode)); similarly, if you are running `yarn lint` in `packages/components`
+
+4. Define component signature
+
+- You can either do it manually [following the documentation](https://typed-ember.gitbook.io/glint/environments/ember/component-signatures#glimmer-components)
```

Comment:
Could it be that it needs to be indented to be rendered as child of `4.`? (unless it's intentional)
```suggestion
    - You can either do it manually [following the documentation](https://typed-ember.gitbook.io/glint/environments/ember/component-signatures#glimmer-components)
```